README.md

```diff
@@ -21,7 +21,8 @@ We run it on Raspbery Pis (ARMv8) as a part of [Umbrel OS](https://github.com/ge
 
 If you're looking to run Umbrel on:
 
-- A Raspberry Pi 3 or 4 (recommended) - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os)
+- A Raspberry Pi 4 with at least 2 Gb RAM - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os) or a 64-bit Linux based system with at least 2 Gb of RAM
```

Comment:
Can you please update this to:

```suggestion
Raspberry Pi 4 with at least 2GB RAM (recommended) - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os)
```

Umbrel OS is not compatible with any "64-bit Linux based system with at least 2 Gb of RAM" so we can't add that. Also, the "recommended" is required since we compare it with "Anything else" (down below).

Reply:
True that Umbrel OS isn't compatible but this is compatible with anything and there are installs where people are just taking this.